src/autoscaler/eventgenerator/config/config.go

```diff
@@ -16,8 +16,10 @@ const (
 	DefaultLoggingLevel                   string        = "info"
 	DefaultPolicyPollerInterval           time.Duration = 40 * time.Second
 	DefaultAggregatorExecuteInterval      time.Duration = 40 * time.Second
+	DefaultSaveInterval                   time.Duration = 30 * time.Second
```

Comment:
the default values looks to long, we need to make the aggregated data available as soon as possible